recipe/meta.yaml

```diff
@@ -11,19 +11,25 @@
 {% set cuda_major_minor = (cuda_major, cuda_minor) %}
 {% endif %}
 
+{% if cross_target_platform is undefined %}
+{% set cross_target_platform = target_platform %}
+{% endif %}
+
 package:
   name: "{{ name }}"
 
 build:
   number: {{ number }}
 {% if cuda_major == 0
-    or (target_platform == "linux-ppc64le" and cuda_major_minor < (10, 2))
-    or (target_platform == "linux-aarch64" and (cuda_major_minor < (11, 0) or arm_variant_type != "sbsa")) %}
+    or (cross_target_platform == "linux-ppc64le" and cuda_major_minor < (10, 2))
+    or (cross_target_platform == "linux-aarch64" and (cuda_major_minor < (11, 0) or arm_variant_type != "sbsa"))
+    or (cross_target_platform != target_platform and cuda_compiler_version != "11.2") %}
```

Comment:
Q: We only support cross-compiling using CUDA 11.2?

Reply:
Yes, we added support for 11.2 only in conda-forge-ci-setup-feedstock